completions/ssh.completion.sh

```diff
@@ -45,7 +45,7 @@ function _omb_completion_ssh {
     fi
   done
   if ((${#config_files[@]} != 0)); then
-    COMPREPLY+=($(compgen -W "$(awk '/^Host/ {for (i=2; i<=NF; i++) print $i}' "${config_files[@]}")" "${options[@]}"))
+    COMPREPLY+=( $(compgen -W "$(awk '/^[ \t]*[Hh][Oo][Ss][Tt][ \t=]+/ {current=""; n=split($0, fields, /[ \t=]+/); for (i=2; i<=n; i++) if (fields[i] && !(fields[i] in hosts)) hosts[fields[i]]=1;} /^[ \t]*[Hh][Oo][Ss][Tt][Nn][Aa][Mm][Ee][ \t=]+/ {n=split($0, fields, /[ \t=]+/); for (i=2; i<=n; i++) if (fields[i] && !(fields[i] in hosts)) hosts[fields[i]]=1;} /^[ \t]*$/ {current="";} END {for (host in hosts) print host;}' "${config_files[@]}")" "${options[@]}") )
```

Comment:
Hmm, I think `n=split($0, fields, /[ \t=]+/);` for `$0 = "  Host myhost"` (with indentation) would produce an empty element as the first field; i.e. it would produce `fields[1] = ""; fields[2] = "Host"; fields[3] = "myhost"`.

Reply:
```suggestion
    COMPREPLY+=($(compgen -W "$(awk '
      sub(/^[ \t]*[Hh][Oo][Ss][Tt]([Nn][Aa][Mm][Ee])?[ \t=]+/, "") {
        n = split($0, fields, /[ \t=]+/);
        for (i = 1; i <= n; i++) if (fields[i] && !visited[fields[i]]++) print fields[i];
      }' "${config_files[@]}")" "${options[@]}") )
```